src/dataviz/updateAnnotationsDW.ts

```diff
@@ -65,6 +65,7 @@ export default async function updateAnnotationsDW(
     }[]
 ) {
     const annotationsWithProps = annotations.map((annotation) => ({
+        ...annotation,
```

Comment:
You probably want it at the end. So you will be overwriting the defaults with your annotation object. Otherwise, it's your annotation object that will be overwritten by defaults. Right?